setup/bindir/cloud-set-guest-password.in

```diff
@@ -26,25 +26,56 @@
 # Modify this line to specify the user (default is root)
 user=root
 
-# Add your DHCP lease folders here
-DHCP_FOLDERS="/var/lib/dhclient/* /var/lib/dhcp3/* /var/lib/dhcp/*"
-PASSWORD_SERVER_PORT=8080
-password_received=0
-file_count=0
-error_count=0
-
-for DHCP_FILE in $DHCP_FOLDERS; do
-	if [ -f $DHCP_FILE ]; then
-		file_count=$((file_count+1))
-		PASSWORD_SERVER_IP=$(grep dhcp-server-identifier $DHCP_FILE | tail -1 | awk '{print $NF}' | tr -d '\;')
-
-		if [ -n "$PASSWORD_SERVER_IP" ]; then
-			logger -t "cloud" "Found password server IP $PASSWORD_SERVER_IP in $DHCP_FILE"
-                        break
-		fi
-	fi
-done
+# Add network provider variables here (default means that interface for network manager is eth0)
+NETPLAN=/etc/netplan
+IFUPDOWN=/etc/network/interfaces
+NETMAN=/etc/sysconfig/network-scripts/ifcfg-eth0
+
+# Add dhcp variables
+PASSWORD_SERVER_PORT=8080                                                                                                                                                                                                                    password_received=0                                                                                                                                                                                                                          error_count=0
```

Comment:
This is line is extremely long, can you make them 3 lines instead?

Reply:
Yes no pb, it is most probably a copy/paste bug, I'll change it right away.